src/runtime/PointerValue.h

```diff
@@ -385,6 +391,12 @@ class PointerValue : public gc {
         return (WeakSetObject*)this;
     }
 
+    GeneratorObject* asGeneratorObject()
+    {
+        ASSERT(isGeneratorObject());
```

Comment:
For readability / State expression explicitly
ASSERT(isGeneratorObject() == true);